WebCryptoAPI/wrapKey_unwrapKey/wrapKey_unwrapKey.https.any.js

```diff
@@ -127,112 +129,117 @@
             formats = ["raw", "jwk"]
         }
 
-        return Promise.all(formats.map(function(fmt) {
-            var originalExport;
-            return subtle.exportKey(fmt, toWrap.key).then(function(exportedKey) {
-                originalExport = exportedKey;
-                const isPossible = wrappingIsPossible(originalExport, wrapper.parameters.name);
-                promise_test(function(test) {
+        formats.map(function(fmt) {
```

Comment:
Do you need a map of the result here?
```suggestion
        formats.forEach(function(fmt) {
```

Reply:
Better, indeed.